maintainers/maintainer-list.nix

```diff
@@ -1067,8 +1067,8 @@
     githubId = 37037;
     name = "Filip Brcic";
     keys = [{
-      longkeyid = "rsa2048/0x0123456789ABCDEF";
-      fingerprint = "AAAA BBBB CCCC DDDD EEEE  FFFF 0000 1111 2222 3333";
+      longkeyid = "rsa4096/0x96577BD3C105EDA4";
```

Comment:
Please squash this commit with the first one and make sure that it is signed with this key. Note that adding keys to the maintainer record is optional.